assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -150,12 +150,10 @@ window.addEventListener( 'load', () => {
 		};
 
 		const openDrawerWithRefresh = () => {
-			miniCartBlock.dataset.isDataOutdated = 'true';
```

Comment:
This change is not directly related to the rest of the PR, but I found out that `isDataOutdated` was defined but never read, so it should be safe to remove it.